meson.build

```diff
@@ -1,5 +1,5 @@
 project('sugar-ext', 'c',
-          version: '2.0.0',
+          version: '4.0.0',
```

Comment:
This isn't actually what we meant when we talked about versioning, seems you misunderstood.

We already talked about keeping the existing version convention, which usually goes with the sugar version released.